_data/data-sponsor.yml

```diff
@@ -0,0 +1,236 @@
+sponsor-home:
+  sponsor-heading: 2022 歌手赛赞助商
+  sponsor-levels:
+    - level-name: platinum
+      level-heading: <br>钻石赞助 虚位以待<br><br>
+
+sponsors_yj:
+  - name: Ginny Lee
+    image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/c_scale,w_200/v1636007705/zgzg-io-website/zgzg-io-website_640_thurn5-c_scale_w_250_woml1t.webp
+    description: 真诚靠谱为客户着想的资深地产团队。
+    website: https://www.bqrealtygroup.com/agents/23-ginny-lee
+  - name: MOOMOO
+    image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/c_scale,w_200/v1636009327/zgzg-io-website/moo_slji2u.png
+    description: 职场人超爱的高效交易APP。
+    website: https://www.moomoo.com/
```

Comment:
Actually, can we remove this `sponsors_yj` list altogether? It doesn't seem to be referenced anywhere.

Reply:
Removed.